plugins/extractit/plugin.spec.yaml

```diff
@@ -247,6 +247,54 @@ actions:
     title: Date Extractor
     description: Extracts all dates from a string or file
     input:
+      date_format:
+        title: Date Format
+        description: Dates matching this format are extracted
+        type: string
+        required: false
+        enum:
```

Comment:
What if the user needs to support multiple date formats? This could be better off as a string array input, referring the user to documentation for supported date format strings.

Reply:
We're adding in support for a wildcard (*) input that will return dates matching any supported format which should help this.

Reply:
Perfect, I think that should be good. I would make one recommendation though, and that is to maybe just make the enum value "All Formats" - that way it's pretty self-documenting and a user won't need to go to the documentation to read about what the `*` symbol does as a date format.

Reply:
Updated with "All Formats" and added support to date_extractor and extract_all actions